opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1152,6 +1152,10 @@ namespace Opm {
         const int np = numPhases();
         well_potentials.resize(nw * np, 0.0);
 
+        if (numWells() == 0) {
```

Comment:
Sorry, but this will not work. First of all I renamed the function to `numLocalWells` and secondly, because of the usage of `DeferrredLogger`  afterwards (new line 1184) which I assume needs all processes to get to that point. One may only return if there are no well at all active globally. For this you need to query `wellsActive()`